validations/quote.test.ts

```diff
@@ -35,33 +37,78 @@ describe('/quote', () => {
       quoteType: 'out',
     })
   }
-  it.each(cases)(
-    'gives quote with quoteId for transfer $quoteType',
-    async ({ quoteParams, quoteType }) => {
+
+  describe.each(cases)('/$quoteType', ({ quoteParams, quoteType }) => {
+    it.each([
```

Comment:
nit: since there are only two cases (get quote with cryptoAmount vs get with fiatAmount), I think this would be more readable and less complex as just 2 different tests, rather than an it.each